llama-index-integrations/vector_stores/llama-index-vector-stores-azureaisearch/llama_index/vector_stores/azureaisearch/base.py

```diff
@@ -713,15 +713,15 @@ def add(
         ids = []
         accumulated_size = 0
         max_size = 16 * 1024 * 1024  # 16MB in bytes
-        max_docs = 1000
+        max_docs = 600  # the limit is 1000 reduced to 600 to avoid the 16MB limit
```

Comment:
Why do we have a max docs parameter? All that matters is the max size right? We could just set the max size to like 10-14MB

Reply:
@logan-markewich the api also limits to 1000 documents/nodes at time, not just by file size